.travis.yml

```diff
@@ -1,16 +1,15 @@
 # This file was generated automatically from conda-smithy. To update this configuration,
-# update the conda-forge.yaml and/or the recipe/meta.yaml.
+# update the conda-forge.yml and/or the recipe/meta.yaml.
 
 language: generic
 
 os: osx
+osx_image: beta-xcode6.1
 
 env:
   matrix:
     
     - CONDA_PY=27
-    - CONDA_PY=34
-    - CONDA_PY=35
```

Comment:
Even the "official" `conda-smithy` is messing this up. Needs some investigation. I will rerender with my hacked version instead.